helmchart/templates/mq/deployment.yaml

```diff
@@ -43,11 +43,12 @@ spec:
           ports:
             - containerPort: {{ .Values.service.mq.sourcePort }}
               protocol: TCP
-          #volumeMounts:
-          #  - name: persistent-storage
-          #    mountPath: /var/lib/rabbitmq
+          volumeMounts:
+            - name: {{ .Values.volumes.tracking.name }}
+              # This folder is available for RabbitMQ messages to be persisted
+              mountPath: {{ .Values.volumes.tracking.mountPath }}
```

Comment:
This folder (`/persist/tracking`) is available for RabbitMQ messages to be persisted. Currently, no messages are configured to persist to this folder.